datasync/downloader/doc.go

```diff
@@ -18,5 +18,25 @@
 // This file is derived from eth/downloader/downloader.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package downloader contains the manual full chain synchronisation.
+/*
+Package downloader contains the manual full chain synchronisation.
+
+How downloader works
+
+The downloader is responsible for synchronizing up-to-date status from the peers connected to it. To do this,
+it downloads "headers", "bodies", and "receipts" in parallel, merges them through the pipeline, and reflect them in the state trie.
```

Comment:
```suggestion
it downloads "headers", "bodies", and "receipts" in parallel, merges them through the pipeline, and reflects them in the state trie.
```

Reply:
What does it mean "reflect them in the state trie"?

Reply:
This means that the downloaded data("headers", "bodies", and "receipts") will be reflected in the state trie.